### ISSUE TITLE
I am unable to use libraries

### ISSUE THREAD

**Issue by yah04dev** [LoreBadTime/PyScript-On-Android#2]
I am unable to use libraries offline is there a way to import them offline.

**Comment by LorVC** [comment]
Hi there, i suggest you to firstly start a new Android project from my source repo, then see the FAQ section from the description https://github.com/LoreBadTime/PyScript-On-Android#faq

**Comment by yah04dev** [comment]
Yes i started a new project and saw the FAQ but i am unable to import python modules when i write import adhan in example it writes module adhan not found i tried <py-env> and <py-config> but not working

**Comment by LorVC** [comment]
Modules import can be a little problematic as i haven't tested anything besides a simple Python run, generally speaking you should be able to use modules if you are able to import them from PyScript, try to import them from the html page or from PyScript things. Keep in mind that this it's only an example project that shows that you could run it theoretically. But practically this isn't ready for any real development case.
